src/lib/spinel/spinel.h

```diff
@@ -4898,6 +4898,35 @@ enum
 
     SPINEL_PROP_BORDER_AGENT__END = 0x970,
 
+    SPINEL_PROP_CLI__BEGIN = 0x970,
+
+    /// CLI Command
+    /** Format: `U` - Write only
+     *
+     * This property allows the host to send a command string to the NCP's command-line interface (CLI). The NCP will
+     * execute the command as if it were entered directly into the CLI. The output of the command (if any) will be sent
+     * back to the host via the `SPINEL_PROP_CLI_OUTPUT` property.
```

Comment:
Could the output of the command be sent back to the host via the `CMD_PROP_VALUE_IS ` of the `SPINEL_PROP_CLI_COMMAND `, rather than adding a new property `SPINEL_PROP_CLI_OUTPUT`?

We can refer to the property [SPINEL_PROP_NEST_STREAM_MFG](https://github.com/openthread/openthread/blob/main/src/lib/spinel/spinel.h#L4903). The host uses this property to execute the diag commands on the RCP. It has the similar function of `SPINEL_PROP_CLI_COMMAND`.

Reply:
After thinking on this, I don't see any problems using the same approach for CLI_COMMAND. So let's change to use the same way. But maybe let's rename the property to CLI_STREAM.